packages/validators/src/schemas/v2023_09_28/experience.ts

```diff
@@ -0,0 +1,185 @@
+import { z } from 'zod';
+import { SchemaVersions } from '../schemaVersions';
+
+const uuidKeySchema = z.string().regex(/^[a-zA-Z0-9-_]{1,21}$/);
+const propertyKeySchema = z.string().regex(/^[a-zA-Z0-9-_]{1,64}$/);
```

Comment:
is there a real need for allowing 64 here, or can we go with 21 or 32 here as well?